charts/opentelemetry-collector/values.yaml

```diff
@@ -63,17 +71,17 @@ config:
 # Can be overridden here or for any component independently using the same keys.
 
 image:
-  repository: otel/opentelemetry-collector
-  # repository: otel/opentelemetry-collector-contrib
+  # repository: otel/opentelemetry-collector
+  repository: otel/opentelemetry-collector-contrib
```

Comment:
will logs eventually land in core? / is the contrib image a better default?

Reply:
filelog receiver which is going to be used for scrapping logs from kubernetes env is under development in contrib. I think that it is not going to be included in core soon if anytime

Reply:
I see, thanks

Reply:
@dmitryax any thoughts on what the implications of these change could be? Contrib is a much larger image, do we care about it?

Reply:
The original idea for this chart was is to support only the core collector (like "essential") functionality by default and provide users with an option to use the contrib collector for other use cases. If we can move `filelog` receiver to the core collector eventually, I'm good with temporarily switching it to contrib.

Reply:
If the logging pipeline is unstable, I'd recommend to set `containerLogs.enabled` to `false` by default, and use contrib build only if user sets it to `true`. @tigrannajaryan @naseemkullah thoughts?

Reply:
sgtm, provided the user can still override the image in either case

Reply:
> If the logging pipeline is unstable, I'd recommend to set `containerLogs.enabled` to `false` by default, and use contrib build only if user sets it to `true`. @tigrannajaryan @naseemkullah thoughts?

@dmitryax Is it possible use different builds/images based on a chart setting? I think that's not a bad idea to do temporarily until filelog is moved to the core.

Reply:
> @dmitryax Is it possible use different builds/images based on a chart setting?

It can be a bit tricky, but must be possible by introducing another template for the image and command arguments.

>  I think that's not a bad idea to do temporarily until filelog is moved to the core.

If we will eventually move the `filelog` receiver to the core, I'm good with temporarily bringing contrib image. Let's put a comment saying that if user don't need the logs pipeline they can set the image back to core.

Reply:
The exact set of receivers we will have in the core is currently being discussed. We are not yet sure the full `filelog` will be in the core. Let's assume that maybe it will not happen in the near future. We need a reasonable solution for now that uses `filelog` from contrib.

Reply:
let's see if we can easily image the image (core vs contrib) dependant on `containerLogs.enabled`. if it's becomes too involved, we can keep contrib image by default?

Reply:
> I'd recommend to set `containerLogs.enabled` to `false` by default, and use `contrib` build only if user sets it to true

Hi!
The `containerLogs.enabled=false` by default make sure at this point.
But I think this should change pretty soon, as soon as we confirm this is working.
That's because gathering logs from the k8s environment is one of the most obvious things a user would want from this chart IMO.

Reply:
While that may be true for other users, today as a user of this chart in production I would disable k8s container logging as that is covered by my providers managed service, I use the collector exclusively for trace data collection.

Reply:
I agree that we shouldn't enable any functionality that isn't part of the stable core functionality

Reply:
I have disabled container logs by default and reverted to using core collector image in https://github.com/open-telemetry/opentelemetry-helm-charts/pull/36/commits/f005dde14bfdd2971aad69a3c0adbe3233a78a97.